### PR TITLE
Stick it to the sidebar

### LINKED ISSUE #1337: Re-stickify sidebar
The sidenav lost its sticky with the redesign, partly because it seems like the nav no longer fits on some screens. We should re-stick it and address the design concerns as needed.

### This task is done when...
 - [x] We have a decision on how the nav should look and function. (For example, we've adjusted the padding to make things fit on a single screen, or we've made the nav wide enough to accommodate longer text on a single line.)
 - [x] The sidebar sticks.
 - [x] The stickiness works across all browsers.

### PR DESCRIPTION
**Pull request by hbillings**
Closes #1337, the best issue number ever. Re-introduces the sticky sidebar.

### This pull request changes...
- Sticks the sidebar.
- Adds [StickyBits](https://www.npmjs.com/package/stickybits) polyfill for older browser support (StickyBits will check to see if it's needed before it starts listening to scroll events)

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [x] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This feature is done when...
- [x] Design has approved the experience
- [x] Product has approved the experience

### CODE REVIEW

**Comment by hbillings** [comment]
@jameshupp, this is ready for you!

File: web/src/containers/Sidebar.js [hunk 1]
```diff
@@ -13,6 +14,10 @@ import { selectActivitiesSidebar } from '../reducers/activities.selectors';
 import { selectActiveSection } from '../reducers/navigation';
 
 class Sidebar extends Component {
+  componentDidMount() {
+    stickybits('.site-sidebar__sticky');
+  }
+
```
Comment (hbillings) [inline web/src/containers/Sidebar.js]: I think this works? We'll see if @jeromeleecms gets a sticky sidebar on IE.

File: web/src/containers/Sidebar.js [hunk 2]
```diff
@@ -206,7 +211,7 @@ class Sidebar extends Component {
                 {t('title', { year: '2018' })}
               </h1>
             </div>
-            <VerticalNav selectedId={activeSection} items={links} />
+            <VerticalNav selectedId={activeSection || "apd-state-profile-overview"} items={links} />
```
Comment (hbillings) [inline web/src/containers/Sidebar.js]: With the new active item highlighting, there was no active item on page load, which caused a weird jump as the font bolded for the first time. That seems to be the only time there's no highlighted item, so this just sets the active item to the first section if `activeSection` is `null`.

**Comment by hbillings** [comment]
@mgwalker I'm not sure why this `check docker-compose image tags` test is failing -- it says something about checksums not matching but I don't really know what's wrong. :-/

**Comment by mgwalker** [comment]
If you run `npm run postinstall` in the web directory, it should update the `docker-compose` file and fix that.

The `docker-compose` file creates images and tags them with the hashes of the `package-lock.json` files.  There's a test to make sure the file hash and the image tags stay in sync.  If you install something new but the `docker-compose` file doesn't get updated, they'll be out of sync.  The `postinstall` npm script is supposed to keep them in sync but apparently sometimes it doesn't run right - and it _definitely_ doesn't run right if you do your `npm install` inside the docker container.

**Comment by hbillings** [comment]
@mgwalker 🙌 No, that's what I love about stickybits -- it checks for `position: sticky` support, and quits if it's there. Most of the React-ready libs I looked at just appended a scroll listener regardless of whether a polyfill was necessary, which seemed like a potential performance issue.

**Comment by jameshupp** [comment]
Same page as Nick: everything is awesome as long as there's also a behavior to make sure the active nav item is visible on screen. If auto-collapse is the easiest way to do that, let's do it! This is very cool!

**Pull request merged**